schemaorg.json

```diff
@@ -0,0 +1,100 @@
+{
+  "@context": "http://schema.org",
+  "@graph": [
```

Comment:
`"@graph": [ {} ` shouldn't be necessary either.  This says it is a collection (graph) made up a list (`[]`) of nodes (`{}`,  SoftwareSourceCode nodes in this case), but in reality it really has only one node, so you can jump right into `"type": "SoftwareSourceCode"` following the `@context` declaration.  

 If you're generating this with a frame, I think  if you add `"@type": "SoftwareSourceCode"` when you frame the JSON-LD, it will see there's only one object of that type and it will return this for you without the `@graph`. 

(Like the other comment, this is not technically incorrect, it is merely more verbose.  which kinda doesn't matter since a computer will be reading it but anyway....